lib/timers.js

```diff
@@ -278,6 +278,11 @@ function listOnTimeout(list, now) {
       return;
     }
 
+    if (ran)
+      runNextTicks();
+    else
+      ran = true;
```

Comment:
Ditto on comments here and above where we use the same pattern, easy to misunderstand